muxplore/theme.c

```diff
@@ -353,14 +353,6 @@ void apply_theme() {
                                   LV_PART_MAIN | LV_STATE_DEFAULT);
     }
 
-    struct small content_pad_right_element[] = {
```

Comment:
now that the content is left aligned setting a width here would prevent you from being able to right align the content.  Instead the width is set on list item itself in main.c